.gitignore

```diff
@@ -9,6 +9,10 @@ db.sqlite3
 media
 staticfiles
 .vscode
+notice_files
+profile_photos
```

Comment:
Why put these files in `.gitignore`? If these are just some files you are creating on your local to do some tests, you don't need to put these in `.gitignore`, just don't stage those files while committing.